crates/ruff_python_formatter/src/preview.rs

```diff
@@ -81,3 +81,8 @@ pub(crate) const fn is_multiline_string_handling_enabled(context: &PyFormatConte
 pub(crate) const fn is_format_module_docstring_enabled(context: &PyFormatContext) -> bool {
     context.is_preview()
 }
+
+/// Returns `true` if the [`PEP 701`](https://github.com/astral-sh/ruff/issues/7594) preview style is enabled.
+pub(crate) fn is_pep_701_enabled(context: &PyFormatContext) -> bool {
```

Comment:
Nit: I would prefer if we call this `fstring_formatting`. Not everyone knows all pep's by heart ;)

Reply:
> Does this gate the entire f string formatting or only the multiline fstring formatting?

The entire f-string formatting. If not enabled, it'll fallback to the previous behavior of normalizing the f-string.